docs/specs/moniker.yml

```diff
@@ -41,6 +41,54 @@ outputs:
       "monikers": []
     }
 ---
+# Apply moniker range setting in the config and yaml header to markdown file's output path and output content
```

Comment:
>Apply moniker r [](start = 2, length = 15)

add test case that monikers is not defined in config but in files

Reply:
and also moniker range in filer is bigger than config

---
In reply to: [232915502](https://github.com/dotnet/docfx/pull/3610#discussion_r232915502) [](ancestors = 232915502)